s3/pkg/service.go

```diff
@@ -406,6 +406,16 @@ func (b *s3Service) DeleteObject(ctx context.Context, in *pb.DeleteObjectInput,
 	return nil
 }
 
+func (b *s3Service) RestoreObject(ctx context.Context, in *pb.RestoreObjectInput, out *pb.BaseResponse) error {
+	log.Logf("Restore object is called in s3 service.")
+	/*
+		restoreObjectInput := pb.GetObjectInput{Bucket:in.Bucket, Key:in.Key}
+		object := pb.Object{}
+		err := db.DbAdapter.RestoreObject(&restoreObjectInput)
+	*/
+	return nil
```

Comment:
remove it

Reply:
We need a flag in database to show the object is in restoring process, otherwise the end user cannot know the restore status.

Reply:
Okay I will update that